src/pages/recurring-expenses/create/Create.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useTitle } from 'common/hooks/useTitle';
+import { Default } from 'components/layouts/Default';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Details } from './components/Details';
+import { Notes } from './components/Notes';
+import { AdditionalInfo } from './components/AdditionalInfo';
+import { request } from 'common/helpers/request';
+import { endpoint } from 'common/helpers';
+import { toast } from 'common/helpers/toast/toast';
+import { useNavigate } from 'react-router-dom';
+import { GenericSingleResourceResponse } from 'common/interfaces/generic-api-response';
+import { TaxSettings } from './components/Taxes';
+import { ValidationBag } from 'common/interfaces/validation-bag';
+import { AxiosError } from 'axios';
+import { route } from 'common/helpers/route';
+import { RecurringExpense } from 'common/interfaces/recurring-expenses';
+import { useBlankRecurringExpenseQuery } from '../common/queries';
+
+
+export function Create() {
+  const [t] = useTranslation();
+
+  const navigate = useNavigate();
+
+  const { documentTitle } = useTitle('new_recurring_expense');
+  const { data } = useBlankRecurringExpenseQuery();
+
+  const pages = [
+    { name: t('recurring_expenses'), href: '/recurring_expenses' },
+    { name: t('new_recurring_expense'), href: 'recurring_expenses/create' },
+  ];
+
+  const [expense, setExpense] = useState<RecurringExpense>();
```

Comment:
I believe you want to extract this to Jotai atom, so it can be created outside of component (ie. cloning).

Reply:
Jotai atom created outside of the component, and used for recurring expense creation